Remove service catalog (#1064)
kyma-project/busola: core-ui/src/service-catalog-ui/delete-instructions.md

> While removing, those can be uninstalled together with service catalog

Reviewer: ```suggestion
You can uninstall the following elements together with Service Catalog:
```
